pom.xml

```diff
@@ -113,6 +114,15 @@
       <artifactId>netty-codec-haproxy</artifactId>
       <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>io.netty.incubator</groupId>
+      <artifactId>netty-incubator-transport-native-io_uring</artifactId>
+      <!-- TODO this should be handled on https://github.com/vert-x3/vertx-dependencies -->
```

Comment:
@vietj what's your suggestion here?

Reply:
yes it should be handled here indeed

Reply:
> yes it should be handled here indeed

you mean https://github.com/vert-x3/vertx-dependencies?
If yes, I suppose I should create a separate linked issue  + PR there too (and vertx parent too)